<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorControllerConfigWindow</class>
 <widget class="QDialog" name="MotorControllerConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controller Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableParams">
     <property name="columnCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnLoadFile">
       <property name="text">
        <string>Load Definition File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
